pkg/tracing/tracing.go

```diff
@@ -125,8 +124,11 @@ func Attribute(k string, v any) attribute.KeyValue {
 	return keyValue(k, v)
 }
 
-// HTTPStatusCodeAttributes generates attributes of the HTTP namespace as specified by the OpenTelemetry
-// specification for a span.
+// HTTPStatusCodeAttributes generates HTTP response status code attributes
+// as specified by the current OpenTelemetry semantic conventions.
 func HTTPStatusCodeAttributes(code int) []attribute.KeyValue {
```

Comment:
Looks like the only use of this function is in the nydus snapshotter; https://github.com/containerd/nydus-snapshotter/blob/eda72f05cd118853c2ea7a2fc3ce206ce97e31b2/pkg/remote/remotes/docker/resolver.go#L598-L600


Maybe we should consider deprecating it 🤔